README.md

```diff
@@ -71,6 +71,37 @@ doctrine:
                     x_reconnect_attempts: 3
 ```
 
+An example of configuration on Symfony 3/4 projects using url instead of separate parameters.
+
+#### Warning:
+The url **MUST** be schemaless. An `InvalidArgumentException` will be thrown otherwise.
+```
+Facile\DoctrineMySQLComeBack\Doctrine\DBAL\Connection needs a driver that implements ServerGoneAwayExceptionsAwareInterface 
+```
+
+```yaml
+# Doctrine example Configuration
+parameters:
+    # Adds a fallback DATABASE_URL if the env var is not set.
```

Comment:
This is wrong. If the cache warmup triggers a DB connection in Doctrine it's not this lib's fault, but it's due to the fact that you didn't specify a DB version/charset in the connection, so Doctrine is forced to connect to it to obtain those.

Reply:
@samvdb 
may you change the PR accordingly with @Jean85 suggestion?
should be really fast, then we'll merge this.
thanks,

Reply:
@samvdb any update on this?